packages/firestore/src/core/firestore_client.ts

```diff
@@ -41,7 +49,7 @@ import { debug } from '../util/log';
 import { Deferred } from '../util/promise';
 
 import { DatabaseId, DatabaseInfo } from './database_info';
-import { Query } from './query';
+import { Query as InternalQuery } from './query';
```

Comment:
I think this type alias is no longer necessary? You can just import it as Query?